### PR TITLE
set up infra and unit test

### PR DESCRIPTION
**Pull request by moody6520**

### CODE REVIEW

File: felucca/common/task.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
+class Task(object):
+    """Task Object
+    Task object represent a pharos executable task 
+    """
+
+    def __init__(self, task_executable_file, tool_type, command_line_input):
+        self.task_executable_file = task_executable_file
```
Comment (xkbt777) [inline felucca/common/task.py:7]: Why task_executable_file is public field?

File: felucca/common/task.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    def set_result(self, output = None, log = None, stdout = None):
+        """Set the result for finished task
+        The execution manager will get three types of result
+
+        Args:
+            output: the output file in json format
+            log: the log file in txt format
+            stdout: the stdout in string format
+        """
+        self.output = output
```
Comment (xkbt777) [inline felucca/common/task.py:65]: Why use self.output instead of self.__output?

File: felucca/common/task.py (hunk 1 above)
Comment (moody6520) [reply felucca/common/task.py]: Solved.

File: felucca/common/task.py (hunk 2 above)
Comment (moody6520) [reply felucca/common/task.py:77]: Solved.

**Pull request merged**